Automattic/wp-calypso
# Launchpad: Add Dismiss launchpad option

Related to #85557
Depends on https://github.com/Automattic/jetpack/pull/34889


## Proposed Changes
* Introduce the Dismiss settings menu only when the launchpad is dismissible
This PR introduces the concept of temporary and permanent launchpad dismiss. Temporary dismiss is based on time, so the frontend is setting a time in the future to describe when the launchpad will be available again. 
The permanent dismiss is based on the already existent 'is_dimissed' flag. 

![image](https://github.com/Automattic/wp-calypso/assets/38718/0abf59f5-62bd-4e24-af29-4877ad710501)


## Testing Instructions
* Set your sandbox with the PR https://github.com/Automattic/jetpack/pull/34889
* Set the build tasklist as dismissible, similar to this change https://github.com/Automattic/jetpack/pull/34839/files#diff-7a7ab71e45292d18195f8de5e1f0338aa5232291425fa0bbe9a21afa65bca23fR273
* Load the site home home/<YOUR_SITE>
* Check if the 3 dots button is rendered
* Choose one of the temporary options following the tip above 
* Check the last option (forever) 


## Pre-merge Checklist
